aten/src/ATen/functorch/BatchRulesDecompositions.cpp

```diff
@@ -168,7 +168,7 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   OP_DECOMPOSE2(movedim, int);
   OP_DECOMPOSE(msort);
   OP_DECOMPOSE(mT);
-  OP_DECOMPOSE(narrow);
+  m.impl("narrow", native::narrow_symint);
```

Comment:
@ezyang @Chillee i'm not sure what this actually does.  It compiles but i'm not happy with the fact that it only handles one of the 2 overloads, and i didn't figure out the right incantation to try to handle both of them.

Reply:
I don't think it matters too much. Basically, we're just registering vmap to decompose compositeimplicitautograd ops.

Reply:
this is fine, I think only hitting the non-Tensor overload is fine, esp if no ci fails. I think technically probably the tensor overload should get a decompose too but I'll file this as preexisting condition